### DISCUSSION TITLE
How long does shields badge reflect new package information in Packagist?

### DISCUSSION THREAD

**Discussion by wadakatu** [badges/shields#7546] [Q&A] [answered]
A couple days ago, I added new package in Packagist.
Then, I put a shields badge that shows the version of my package on README file.

However, it says "Not Found".

Does anyone know how long I should wait approximately until the badge works ?

**Comment by calebcartwright** [comment, marked as answer]
Sounds similar to/duplicate of #6509.

However, it's difficult to discuss these things in the abstract, would be helpful if you could share the badge url you're trying to use

**Reply by wadakatu** [reply]
@calebcartwright 
Thank you for your message.

My badge and package url are like below.

Package URL: https://packagist.org/packages/wadakatu/laravel-factory-refactor
Badge URL: https://img.shields.io/packagist/v/wadakatu/laravel-factory-refactor ![Packagist Version](https://img.shields.io/packagist/v/wadakatu/laravel-factory-refactor)

**Reply by wadakatu** [reply]
In my case, it took nearly two weeks to reflect my badge info.

Hope it helps.
